CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package depthimage_to_laserscan
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.2.1 (2019-03-29)
```

Comment:
Let's not modify the CHANGELOG with this PR; that will get done during release time.